docs/guide/overview-of-event-sources.md

```diff
@@ -140,3 +147,47 @@ functions:
           topic_name: aggregate
           display_name: Data aggregation pipeline
 ```
+
+### Kinesis Streams
+
+Currently there's no native support for Kinesis Streams ([we need your feedback](https://github.com/serverless/serverless/issues/1608))
+but you can use custom provider resources to setup the mapping:
+
+```yml
+# serverless.yml
+
+resources
+  Resources:
+    mapping:
+      Type: AWS::Lambda::EventSourceMapping
+      Properties:
+        BatchSize: 10
+        EventSourceArn: "arn:aws:kinesis:<region>:<aws-account-id>:stream/<stream-name>"
```

Comment:
I think most users would setup the stream using custom resources as well. In that case they wouldn't have the arn available and they'd use `Ref` in that case. We should at least have a comment or note that mentions that case when they're creating the stream via Serverless custom resources.